.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-* @devinleighsmith @asanchezr @FuriousLlama @stairaku @Maheshbabu28 @PryancaJSharma
+* @devinleighsmith @asanchezr @FuriousLlama @stairaku @Maheshbabu28 @PryancaJSharma @eddherrera
```

Comment:
![image](https://user-images.githubusercontent.com/24322224/209745113-f1926e95-97e6-4186-a04d-05b266b571b5.png)

Reply:
he does exist but likely doesn't have write access yet until James Payer is back.